ansible/configs/kni-osp/pre_software.yml

```diff
@@ -0,0 +1,108 @@
+---
+- name: Step 003 - Create env key
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: false
+  tags:
+  - step003
+  - generate_env_keys
+  tasks:
+  - name: Generate SSH keys
```

Comment:
tasks on 11-27 are now a role `infra-local-create-ssh_key`.

Also, this is a good candidate to move to pre-infra